### PR TITLE
Update config_vsc.py, hortense support

### PR DESCRIPTION
**Pull request by Lewih**
Hortense access flag is selected according to the user running the test suite

### CODE REVIEW

File: config_vsc.py [hunk 1]
```diff
@@ -24,6 +26,20 @@
 # vsc group
 kul_account_string_tier2 = '-A lpt2_vsc_test_suite'
 
+# Specify hortense access flag in order to run jobs
+# Flag is selected according to user group
+groups = os.popen('groups').read().strip().split()
```
Comment (smoors) [inline config_vsc.py:31]:
```suggestion
groups = [x.gr_name for x in grp.getgrall()]
```

File: config_vsc.py [hunk 2]
```diff
@@ -24,6 +26,20 @@
...
+if 'astaff' in groups:
+    admingroup = 'astaff'
+elif 'bstaff' in groups:
+    admingroup = 'bstaff'
```
Comment (smoors) [inline config_vsc.py:35]:
```suggestion
elif 'badmin' in groups:
    admingroup = 'badmin'
```

File: config_vsc.py (hunk 1 above)
Comment (smoors) [reply config_vsc.py:31]: oh you're right, my bad.
then I prefer doing it like this:

```python
hortense_access_flag = ''
groups = [grp.getgrgid(x).gr_name for x in os.getgroups()]
for admingroup in ['astaff', 'badmin', 'gadminforever', 'lstaff']:
    if admingroup in groups:
        hortense_access_flag = f'-A {admingroup}'
        break
```

Comment (Lewih) [reply config_vsc.py:31]: looks better.

Comment (Lewih) [reply config_vsc.py]: committed

File: config_vsc.py [hunk 3]
```diff
@@ -1,3 +1,5 @@
+import os
```
Comment (smoors) [inline config_vsc.py:1]:
```suggestion
import grp
import os
```

File: config_vsc.py [hunk 4]
```diff
@@ -24,6 +27,16 @@
 # vsc group
 kul_account_string_tier2 = '-A lpt2_vsc_test_suite'
 
+# Specify hortense access flag in order to run jobs
+# Flag is selected according to user group
+hortense_access_flag = ''
+groups = [grp.getgrgid(x).gr_name for x in os.getgroups()]
+for admingroup in ['astaff', 'badmin', 'gadminforever', 'lstaff']:
```
Comment (boegel) [inline config_vsc.py:34]: For Leuven, it should be `l_sysadmin`

Comment (Lewih) [reply config_vsc.py]: added

**Review by smoors — APPROVED** [review]
lgtm

**Pull request merged**